pants.toml

```diff
@@ -2,15 +2,11 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 [GLOBAL]
-pants_version = "1.30.0"
-v1 =  false  # Turn off the v1 execution engine.
-v2 = true  # Enable the v2 execution engine.
+pants_version = "2.0.0.dev2"
```

Comment:
https://github.com/pantsbuild/pants/issues/10425 is now fixed, and [2.0.0.dev5](https://pypi.org/project/pantsbuild.pants/2.0.0.dev5/) is released!

Reply:
Brilliant, works like a charm. That's an excellent response time everyone 👍

Reply:
Yay! We added a change to Pants's internal setup so that we would have detected this case much sooner.